src/UsgsAstroLsSensorModel.cpp

```diff
@@ -1681,20 +1682,12 @@ void UsgsAstroLsSensorModel::computeDistortedFocalPlaneCoordinates(const double&
 void UsgsAstroLsSensorModel::computeUndistortedFocalPlaneCoordinates(const double &distortedFocalPlaneX, const double& distortedFocalPlaneY, double& undistortedFocalPlaneX, double& undistortedFocalPlaneY) const{

```

Comment:
I don't see any point to having this function any more (`UsgsAstroLsSensorModel::computeUndistortedFocalPlaneCoordinates`). I think it can be deleted and replaced where it is called in the code with the call to `removeDistortion` that you added in the body of the function.

Reply:
How do we want to handle modal distortion? Transverse and Radial distortion methods have different signatures, so we're going to need some sort of a "switch" statement.

Reply:
It looks like modal distortion isn't handled right now at the "reading in the json" level. Agreed about needing some kind of switch statement and figuring out where it belongs.